tests/acceptance/seller/InternalTransferCest.php

```diff
@@ -0,0 +1,85 @@
+<?php 
+
+namespace hipanel\modules\finance\tests\acceptance\seller;
+
+use hipanel\helpers\Url;
+use Codeception\Example;
+use hipanel\tests\_support\Step\Acceptance\Seller;
+use hipanel\tests\_support\Page\Widget\Input\Input;
+use hipanel\tests\_support\Page\Widget\Input\Select2;
+use hipanel\modules\finance\tests\_support\Page\transfer\Create as TransferCreate;
+use hipanel\modules\finance\tests\_support\Page\bill\Create as BillCreate;
+use hipanel\modules\finance\tests\_support\Page\transfer\Index;
+
+class InternalTransferCest
+{
+    private $transferCreate;
+    private $billCreate;
+    private $index;
```

Comment:
```suggestion
    private Index $index;
```
мав на увазі отак зробить